### PR TITLE
Setup & Docker updates

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
CuPy, turboJPEG and rpp updates

### CODE REVIEW

**Comment by LakshmiKumar23** [comment]
@hansely Please approve if this looks ok

**Comment by LakshmiKumar23** [comment]
@kiritigowda Ready to be merged.

File: docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install initramfs-tools libnuma-de
         wget ${ROCM_INSTALLER_REPO} && \
         sudo apt-get install -y ./${ROCM_INSTALLER_PACKAGE} && \
         sudo apt-get update -y && \
-        sudo amdgpu-install -y --usecase=graphics,rocm
+        sudo amdgpu-install -y --usecase=graphics,rocm --no-32
```
Comment (rrawther) [inline docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile:23]: probably we can use --usecase=multimedia,rocm instead?

**Review by rrawther — COMMENTED** [review]
@LakshmiKumar23 : added comment

File: docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile (hunk 1 above)
Comment (LakshmiKumar23) [reply docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile:23]: @rrawther The usecase multimediasdk was not tested thoroughly on rocm6.0 and these dockers are currently on that version of rocm. We can switch to multimediasdk in the next version

**Comment by LakshmiKumar23** [comment]
@rrawther Can you approve this if it looks okay?

**Review by rrawther — APPROVED** [review]

**Comment by LakshmiKumar23** [comment]
@kiritigowda can we merge this?

**Pull request merged**